apps/angular-console/src/app/app.module.ts

```diff
@@ -88,7 +89,12 @@ export function initApollo(
         {
           path: '',
           children: workspaceRoutes,
-          canActivateChild: [CancelCommandGuard]
+          canActivateChild: [IsNodeJsInstalledGuard, CancelCommandGuard]
+        },
+        {
+          path: 'install-nodejs',
+          loadChildren:
+            '@angular-console/feature-install-node-js#FeatureInstallNodeJsModule'
         },
         {
           path: 'support',
```

Comment:
We should probably make support lazy as well in a later PR

Reply:
Agreed, should speed up boot up time a bit more (not that Jason hasn't already solved that)